apps/api/tests/auth/test_internal_services.py

```
@@ -0,0 +1,364 @@
+            },
+        )
+
+        self.assertEqual(outcome, 'supplied')
+        self.assertEqual(emit, {})
+        self.assertEqual(len(db.issued('CREATE (n:ClientCredential')), 1)
+
+    async def test_environment_value_repoints_existing(self) -> None:
+        """A credential the environment names has its hash rewritten."""
+        db = FakeGraph({'SET c.client_secret_hash': [{'c': 'credential'}]})
+
+        outcome, emit = await internal_services._ensure_client_credential(
```

Ruff (0.16.0)

[warning] Unpacked variable `emit` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)